### ISSUE TITLE
Join forces with storybook_flutter?

### ISSUE THREAD

**Issue by MuathAmer** [erickzanardo/dashbook#22]
Let's make one great flutter dev-tool instead of many!

Hi! first of all, I want to say that I really appreciate the works in this open-source project, I installed it and all other similar packages, and I think it can be great if the work is unified into a single tool which can become popular and a standard for flutter development. 

I will list all the projects that I found here:
- https://github.com/erickzanardo/dashbook
- https://github.com/ookami-kb/storybook_flutter
- https://github.com/ilikerobots/storyboard

These 3 have the same goal, a storybookjs inspired package to showcase flutter widgets.

But there are also these projects that if combined with a flutter storybook, can really become a standard tool for every flutter developer:
- https://github.com/jamesblasco/flutter_preview (preview any widget in vs code)
- https://github.com/aloisdeniel/flutter_device_preview (emulate any device size)
- https://github.com/rodydavis/storyboard (see all routes at once)

I am pasting this same issue to all of these projects, if you don't want it, feel free to close it :D

Tell me what you think!

**Comment by erickzanardo** [comment]
I am all in favor of joining forces with other devs that made similar tools.

Just to give you more context on why I created Dashbook.

When I felt the need of a package like this, I did some research and dicovered `storyboard`, which is an awesome package, but back then I was very used to the storybookjs API and storyboard had an API a little different, which is not a bad API at all, I really just wanted something similar to what I was used to use.

On Dashbook I tried to keep it similar to StorybookJS API and that is still what is the idea of this package.

Anyhow, I am open to collaboration with the authors of the other packages if they have interest as well.
